### ISSUE TITLE
UserWarning and RuntimeWarning during Topaz Model Training

### ISSUE THREAD

**Issue by moranchu** [tbepler/topaz#209]
I am encountering several warnings during the training of a model using Topaz in **relion 5.0 beta4**. The training process seems to proceed normally, but I am concerned about the potential impact of these warnings on the final results. Below is the complete log of the training session:

Will use topaz for training a model 
Written out list of input training coordinates: AutoPick/job079/input_training_coords.star
Setting topaz downscale factor to 6 (assuming resnet8 model and 2*particle_diameter receptive box)
Setting topaz radius to 4 downscaled pixels (based on 25% of particle_diameter/2)
Using GPU device 0
Training with 337 picks in test set; and 729 picks in work set
By setting aside 3 micrographs for the test set 
Loading model: resnet8
Model parameters: units=32, dropout=0.0, bn=on
Loading pretrained model: resnet8_u32
Receptive field: 71
Using device=0 with cuda=True
Loaded 7 training micrographs with 729 labeled particles
Loaded 3 test micrographs with 337 labeled particles
source	split	p_observed	num_positive_regions	total_regions
0	train	0.0129	35721	2764314
0	test	0.0139	16513	1184706
Specified expected number of particle per micrograph = 600.0
With radius = 4
Setting pi = 0.07444885060090858
minibatch_size=256, epoch_size=1000, num_epochs=10

UserWarning: The given NumPy array is not writable, and PyTorch does not support non-writable tensors. This means writing to this tensor will result in undefined behavior. You may want to copy the array to protect its data or make it writable before converting it to a tensor. This type of warning will be suppressed for the rest of this program. (Triggered internally at ../torch/csrc/utils/tensor_numpy.cpp:206.)
UserWarning: The given NumPy array is not writable, and PyTorch does not support non-writable tensors. This means writing to this tensor will result in undefined behavior. You may want to copy the array to protect its data or make it writable before converting it to a tensor. This type of warning will be suppressed for the rest of this program. (Triggered internally at ../torch/csrc/utils/tensor_numpy.cpp:206.)
RuntimeWarning: overflow encountered in exp

**Questions:**
Are these warnings expected, or do they indicate a potential issue with the training process?
Could these warnings affect the accuracy or reliability of the trained model?
If these warnings are problematic, what steps can be taken to resolve them?
I would appreciate any insights or guidance on how to handle these warnings effectively. Thank you!

**Environment:**
relion 5.0 beta4

**Comment by tbepler** [comment]
You can safely ignore these warnings
